frontend/components/SidePanelContent/_styles.scss

```diff
@@ -8,6 +8,11 @@
   // need relative positioning for a close icon that is on some sidebars have
   position: relative;
   animation: fade-in 250ms ease-out;
+  display: none;
+
+  @media (min-width: $break-sm) {
+    display: block;
+  }
```

Comment:
Same mobile-first pattern where the default is to be hidden, and progressively at 768px it becomes visible.